vcpkg/users/versioning.md

```diff
@@ -112,6 +118,9 @@ Examples:
 - `2021-01-01#20` < `2021-01-01.1`
 - `windows#7` < `windows#8`
 
+[!NOTE]
+When making changes to a port, it's important to ensure that the [version database](versioning.concepts.md#acquiring-port-versions) is updated. To facilitate this, you can use the [`vcpkg x-add-version`](..\commands\add-version.md) command.

```

Comment:
I don't think the `port-versions` section of the Versioning Reference is the right place for this.